api_tests/preprints/views/test_preprint_versions.py

```diff
@@ -0,0 +1,78 @@
+from api.base.settings.defaults import API_BASE
+from osf.models import Preprint
+from osf.utils import permissions
+from osf_tests.factories import (
+    ProjectFactory,
+    PreprintFactory,
+    AuthUserFactory,
+)
+from tests.base import ApiTestCase
+
+from django.db.models.fields import Field
+
+
+class TestPreprintVersion(ApiTestCase):
+
+    def setUp(self):
+        super().setUp()
+        self.user = AuthUserFactory()
+
+        self.preprint = PreprintFactory(creator=self.user)
+
+        self.project = ProjectFactory(creator=self.user)
+
+        self.url = f'/{API_BASE}preprints/{self.preprint._id}/versions/'
+
+    def test_create_preprint_version(self):
+        res = self.app.post_json_api(self.url, auth=self.user.auth)
+
+        assert res.status_code == 201
```

Comment:
We should return 200 since version creation returns the object instead of a reference.

We should also check a few key properties (those we've added/updated in the serializer) in the response. But I see you have them in separate tests below, so no worries.